ci/run-bridge.sh

```diff
@@ -72,7 +82,15 @@ main() {
     --rpc.addr 0.0.0.0 \
     --core.ip validator \
     --keyring.keyname "$NODE_NAME" \
-    --p2p.network "$P2P_NETWORK"
+    --p2p.network "$P2P_NETWORK" &
+
+  # each node without SKIP_AUTH connects to the one with, so that bridges can discover eachother
+  if [ ! "$SKIP_AUTH" == "true" ] ; then
+    connect_to_common_bridge
+  fi
+
+  # don't exit until celestia bridge running in the background does
+  wait
```

Comment:
can we put `connect_to_common_bridge` to background before `bridge start` and remove the `wait`?

Reply:
Sure, swapped :+1: